src/coreclr/vm/codeman.h

```diff
@@ -287,6 +296,13 @@ struct CodeHeader
         SUPPORTS_DAC;
         return pRealCodeHeader->nUnwindInfos;
     }
+
+    IsFuncletCache ComputeInitialIsFuncletCacheValue(PCODE codePtr)
```

Comment:
It would be nice for `IsFuncletCache` is kept as a private detail of EECodeInfo.

bool MayHaveFunclets()
{
    return GetNumberOfUnwindInfos() > 1;
}